packages/core/src/components/forms/formGroup.tsx

```diff
@@ -54,14 +54,14 @@ export class FormGroup extends React.PureComponent<IFormGroupProps, {}> {
      *
      * Defaults to `<span class={Classes.TEXT_MUTED}>(required)</span>`.
      */
-    public static DEFAULT_REQUIRED_CONTENT = <span className={Classes.TEXT_MUTED}>(required)</span>;
+    public static DEFAULT_REQUIRED_CONTENT = <span className={Classes.TEXT_MUTED}> (required)</span>;
```

Comment:
this doesn't fix the issue if the user supplies their own required label. instead add `{" "}` in the JSX itself between the two elements (at the end of line 64).